### PR TITLE
[OS Patch] Habana TensorFlow 2.9

### PR DESCRIPTION
**Pull request by junpuf**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by junpuf** [comment]
PASSED sanity/test_ecr_scan.py::test_ecr_enhanced_scan[669063966089.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-training-habana:2.9.1-hpu-py38-synapseai1.6.0-ubuntu20.04-example-pr-2251-2022-09-22-23-25-27]
PASSED sanity/test_ecr_scan.py::test_ecr_enhanced_scan[669063966089.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-training-habana:2.9.1-hpu-py38-synapseai1.6.0-ubuntu20.04-pr-2251-2022-09-22-23-25-27]
FAILED sanity/test_anaconda.py::test_repo_anaconda_not_present[669063966089.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-training-habana:2.9.1-hpu-py38-synapseai1.6.0-ubuntu20.04-pr-2251-2022-09-22-23-25-27]
968 | FAILED sanity/test_anaconda.py::test_repo_anaconda_not_present[669063966089.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-training-habana:2.9.1-hpu-py38-synapseai1.6.0-ubuntu20.04-example-pr-2251-2022-09-22-23-25-27]

**Review by saimidu — APPROVED** [review]
LGTM.

File: habana/tensorflow/training/docker/2.9/py3/synapseai1.5.0/Dockerfile.hpu [hunk 1]
```diff
@@ -81,6 +92,9 @@ RUN apt-get update \
 
 RUN ${PIP} --no-cache-dir install --upgrade \
     pip \
+    oauthlib \
+    # Protobuf 3.18.3, 3.19.5, 3.20.2 and 4.21.6 include a fix for CVE-2022-1941
+    protobuf==3.19.5 \
```
Comment (kace) [inline habana/tensorflow/training/docker/2.9/py3/synapseai1.5.0/Dockerfile.hpu:97]: nit: Did we want to manage these packages at the system level? Does it make sense to install them w/ the other packages starting at #L112?

**Review by kace — COMMENTED** [review]
Single nit. Otherwise LGTM.

File: habana/tensorflow/training/docker/2.9/py3/synapseai1.5.0/Dockerfile.hpu (hunk 1 above)
Comment (junpuf) [reply habana/tensorflow/training/docker/2.9/py3/synapseai1.5.0/Dockerfile.hpu:97]: i think it does make sense to have this one join the ones starting L112 since these are TF related things. I'm ok with either way, if you want to move these to the TF stuff i could make another commit too

Comment (junpuf) [reply habana/tensorflow/training/docker/2.9/py3/synapseai1.5.0/Dockerfile.hpu:97]: let me make a commit anyway since it need to revert the developer_config changes.

**Review by saimidu — APPROVED** [review]
